### DISCUSSION TITLE
[BUG] Cuando se levanta un formulario y dentro ellos tiene un componente multiple choice, el texto sale fuera del formulario

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#200] [4. Bug Reports] [answered]
Hola

Adjunto un video y la foto de lo que estoy indicando. No sé si es un bug o una feature.

**Link video:** https://drive.google.com/file/d/1yVUw08DIbLuJjwI__aS-PQpoCvSp-QU_/view?usp=sharing

Foto
![image](https://user-images.githubusercontent.com/61419471/118874772-428e1380-b8b9-11eb-8349-18debce66bfc.png)

Gracias :>

**Comment by gmoplaza** [comment, marked as answer]
@NicolasPLG es un bug, lo voy a reporta, gracias!
